airbyte-integrations/connectors/destination-bigquery/src/test-integration/java/io/airbyte/integrations/destination/bigquery/typing_deduping/AbstractBigQueryTypingDedupingTest.java

```diff
@@ -96,6 +97,7 @@ protected SqlGenerator getSqlGenerator() {
    * _airbyte_data). Then run a sync using our current version.
    */
   @Test
+  @Disabled("We no longer need to run V2 migrators, this was a safety net during DV2 beta period")
```

Comment:
I'd vote to just delete the test completely at this point

Reply:
👍  I thought it was coming from base but it isn't.